Sources/Amplitude/PrivacyInfo.xcprivacy

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>NSPrivacyCollectedDataTypes</key>
+	<array>
+		<dict>
+			<key>NSPrivacyCollectedDataType</key>
+			<string>NSPrivacyCollectedDataTypeDeviceID</string>
+			<key>NSPrivacyCollectedDataTypeLinked</key>
+			<true/>
+			<key>NSPrivacyCollectedDataTypeTracking</key>
+			<true/>
+			<key>NSPrivacyCollectedDataTypePurposes</key>
+			<array>
+				<string>NSPrivacyCollectedDataTypePurposeAnalytics</string>
+			</array>
+		</dict>
+		<dict>
+			<key>NSPrivacyCollectedDataType</key>
+			<string>NSPrivacyCollectedDataTypePreciseLocation</string>
+			<key>NSPrivacyCollectedDataTypeLinked</key>
+			<true/>
+			<key>NSPrivacyCollectedDataTypeTracking</key>
+			<true/>
+			<key>NSPrivacyCollectedDataTypePurposes</key>
+			<array>
+				<string>NSPrivacyCollectedDataTypePurposeAnalytics</string>
+			</array>
+		</dict>
+	</array>
+	<key>NSPrivacyTrackingDomains</key>
+	<array>
+		<string>https://api2.amplitude.com/2/httpapi</string>
```

Comment:
Do we not send data to the batch endpoint too?

Reply:
I kind of agree that Izaaz to have all [domains](https://github.com/amplitude/Amplitude-Swift/blob/353b334924d05c67a85d403584a065999a778c2b/Sources/Amplitude/Constants.swift#L50-L53) listed here even though not all of them will be used. 

Because if users don't accept the tracking permission, there domains will just be blocked. If users accept the permission, these domains are allowed to send data to. It won't accept customers and users either way. And the pros is that customers don't need to add `NSPrivacyTrackingDomains` by themselves if they config to EU or `useBatch`. 

Added in https://github.com/amplitude/Amplitude-Swift/pull/109/commits/dbc67333e4d977ea3c2d4dab3dc8f5bd8a76c4ec